charts/zesty/values.yaml

```diff
@@ -1,208 +1,903 @@
+  # @schema
   enabled: false
+  # @schema
+  # type: boolean
+  # required: true
+  # @schema
   curV2: false
-  athenaProjectID: ~
-  athenaBucketName: ~
-  athenaRegion: ~
-  athenaDatabase: ~
-  athenaTable: ~
-  athenaWorkgroup: ~
-  athenaCatalog: ~
-
+  # @schema
+  # type: string
+  # required: false
+  # @schema
+  athenaProjectID: ""
+  # @schema
+  # type: string
+  # required: false
+  # @schema
+  athenaBucketName: ""
+  # @schema
+  # type: string
+  # required: false
+  # @schema
+  athenaRegion: ""
+  # @schema
+  # type: string
+  # required: false
+  # @schema
+  athenaDatabase: ""
+  # @schema
+  # type: string
+  # required: false
+  # @schema
+  athenaTable: ""
+  # @schema
+  # type: string
+  # required: false
+  # @schema
+  athenaWorkgroup: ""
+  # @schema
+  # type: string
+  # required: false
+  # @schema
+  athenaCatalog: ""
+
+# @schema
+# type: object
+# required: true
+# @schema
 assumeRole:
+  # @schema
+  # type: boolean
+  # required: true
+  # @schema
   enabled: false
-  masterRoleArn: ~
-  masterExternalID: ~
-  zestyExternalID: ~
-  roleArn: ~
-
+  # @schema
+  # type: string
+  # required: false
+  # @schema
+  masterRoleArn: ""
+  # @schema
+  # type: string
+  # required: false
+  # @schema
+  masterExternalID: ""
+  # @schema
+  # type: string
+  # required: false
+  # @schema
+  zestyExternalID: ""
+  # @schema
+  # type: string
+  # required: false
+  # @schema
+  roleArn: ""
+
+# @schema
+# type: object
+# required: true
+# @schema
 s3override:
+  # @schema
+  # type: boolean
+  # required: true
+  # @schema
   enabled: false
-  bucket: ~
-  region: ~
-
+  # @schema
+  # type: string
+  # required: false
+  # @schema
+  bucket: ""
+  # @schema
+  # type: string
+  # required: false
+  # @schema
+  region: ""
+
+# @schema
+# type: object
+# required: true
+# @schema
 metadata:
+  # @schema
+  # type: boolean
+  # required: true
+  # @schema
   enabled: false
-  provider: ~
-  accountID: ~
-  region: ~
-  clusterName: ~
+  # @schema
+  # type: string
+  # required: false
+  # @schema
+  provider: ""
+  # @schema
+  # type: string
+  # required: false
+  # @schema
+  accountID: ""
+  # @schema
+  # type: string
+  # required: false
+  # @schema
+  region: ""
+  # @schema
+  # type: string
+  # required: false
+  # @schema
+  clusterName: ""
 
+# @schema
+# type: object
+# required: true
+# @schema
 config:
+  # @schema
+  # type: object
+  # required: true
+  # @schema
   manager:
+    # @schema
+    # type: boolean
+    # required: true
+    # @schema
     enable: false
 
+# @schema
+# type: object
+# required: true
+# @schema
 podMinuteMetrics:
+  # @schema
+  # type: boolean
+  # required: true
+  # @schema
   enabled: true
 
+# @schema
+# type: object
+# required: true
+# @schema
 insights:
+  # @schema
+  # type: array
+  # required: false
+  # @schema
   extraEnv: []
 
+  # @schema
+  # type: object
+  # required: false
+  # additionalProperties: true
+  # @schema
   podAnnotations: {}
```

Comment:
You can enforce that the values for all keys will be string using patternproperties.
https://github.com/dadav/helm-schema?tab=readme-ov-file#patternproperties